release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1408,14 +1407,12 @@ This release removes the Prometheus Adapter, which was a Technology Preview feat
 * Previously, the Cluster Version Operator evaluated both the `Available` and `Degraded` parameters when setting the `cluster_operator_up` metric, which caused the `ClusterOperatorDown` alert to be displayed for Operators with `Available=True` or `Degraded=True`, even though `Available=True` did not match the alert description of "has not been available". With this fix, the Cluster Version Operator now ignores the `Degraded` parameter when setting the `cluster_operator_up` metric. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1834551[*BZ#1834551*])
 
 * Previously, when Prometheus was installed on the cluster, important platform topology metrics were not available and a CI error would occur if the installer metric that was generated with the invoker was set to `""`. The possible race condition in which informers were not synced before metrics were served that was causing the error has now been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1871303[*BZ#1871303*])
-  
-* Previously, manifests with multiple tolerations for the same key, such as the Cluster Version Operator's own deployment), would accept only the last entry read and overwrite prior entries. This caused `in-cluster tolerations` to diverge from the manifest's listed tolerations. With this update, the Cluster Version Operator now considers tolerations matching when they are completely equal. This allows the Cluster Version Operator to keep all tolerations present in the manifest for the `in-cluster resource`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1941901[*BZ#1941901*])
-  
 
+* Previously, manifests with multiple tolerations for the same key, such as the Cluster Version Operator's own deployment), would accept only the last entry read and overwrite prior entries. This caused `in-cluster tolerations` to diverge from the manifest's listed tolerations. With this update, the Cluster Version Operator now considers tolerations matching when they are completely equal. This allows the Cluster Version Operator to keep all tolerations present in the manifest for the `in-cluster resource`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1941901[*BZ#1941901*])
```

Comment:
s/deployment)/deployment 
s/Cluster Version Operator's own deployment/Cluster Version Operator's deployment object or `cluster-version-operator` deployment object ??
I'm not sure what _in-cluster tolerations_ are. I don't think it is a term we use in the docs.  Same with _in-cluster resource_.
s/now considers tolerations matching/now considers tolerations to match (or something like that)

Reply:
This one is missing some edits I remember making :upside_down_face: Thanks for the "matching" phrasing help, I was unhappy with that but couldn't come up with better at the time.